pkg/certificate/manager_test.go

```diff
@@ -116,7 +116,7 @@ var _ = Describe("certificate manager", func() {
 	loadCASecret := func(manager *Manager) corev1.Secret {
 		secretKey := types.NamespacedName{
 			Namespace: "default",
-			Name:      expectedMutatingWebhookConfiguration.Name,
+			Name:      expectedMutatingWebhookConfiguration.Name + "-ca",
```

Comment:
can we use `caSecretKey()` here?

Reply:
We don't have the manager here so we can call `caSecretKey`